src/main/java/org/avarion/graves/integration/SimpleClaimSystem.java

```diff
@@ -0,0 +1,40 @@
+package org.avarion.graves.integration;
+
+import fr.xyness.SCS.API.SimpleClaimSystemAPI;
+import fr.xyness.SCS.API.SimpleClaimSystemAPI_Provider;
+import fr.xyness.SCS.Types.Claim;
+import org.bukkit.Location;
+import org.bukkit.entity.Player;
+import org.bukkit.plugin.Plugin;
+
+public final class SimpleClaimSystem {
+
+    private static SimpleClaimSystemAPI scs;
+
+    public SimpleClaimSystem(Plugin simpleClaimSystemPlugin) {
+        SimpleClaimSystemAPI_Provider.initialize((fr.xyness.SCS.SimpleClaimSystem) simpleClaimSystemPlugin);
```

Comment:
Why do an unchecked cast here?
I think it'd be better to accept this type instead of `Plugin`